src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -406,8 +406,18 @@ public function getVersions()
     public function getVersionLog()
     {
         $result = array();
-        $rows = $this->fetchAll(sprintf('SELECT * FROM %s ORDER BY version ASC', $this->getSchemaTableName()));
-        foreach ($rows as $version) {
+
+        switch ($this->options['version_order']) {
+            case \Phinx\Config\Config::VERSION_ORDER_CREATION_TIME:
```

Comment:
Won't that be confusing, having 2 names for the same thing? Since in the end we don't actually use the `version` and `start_time` column names directly (as you can see down there, it's more of a `version -> version ASC` and `start_time -> start_time ASC, version ASC` mapping), it seems to me like adding confusion for no real gain?

I do think it looks indeed nicer if I rename the valid config option values from `creation-time` and `start-time` to `creation` and `execution` like you said, makes it clearer for the end-user